text/0096-rename-tasks-evaluations-hooks.md

```diff
@@ -0,0 +1,47 @@
+# Group Keptn Tasks and Evaluations as Hooks
+
+Proposal to change the way the project refers to Keptn Tasks and Evaluations. Instead, refer to both as Keptn Hooks.
+
+## Motivation
+
+* Make it simpler for a user to understand the core concepts of Keptn Lifecycle Toolkit
+
+## Explanation
+
+Hooks are a fairly standard and well understood software mechanism. So rather than referring to tasks and evaluations - then attempting to explain the difference.
+
+Instead we start referring to Keptn hooks. Keptn hooks are a way to interject into the lifecycle of a running Keptn process.
+
+With this description, the explanation becomes easy. There are two possible Keptn hooks:
+
+- Evaluations
+- Tasks
+
+Keptn hooks can be configured to fire at any of these times:
+
+- Pre-deployment at a workload level
+- Pre-deployment at an application level
+- Pre-deployment at a workload level
+- Post-deployment at a workload level
```

Comment:
I may be missing something but it seems like this requires changing the names of the annotations and sort of blurring the distinction between evaluations and tasks, at least at the configuration level.  Do we have evaluations and tasks that are executed only at the KeptnApp level?  I thought all the evaluations and tasks defined for the individual workloads were executed for any KeptnApp that includes those workloads.

I'm looking forward from comments from people with more knowledge!